site/Head.tsx

```diff
@@ -1,5 +1,33 @@
 import React from "react"
 import { VITE_ASSET_SITE_ENTRY, viteAssets } from "./viteUtils.js"
+import { GOOGLE_TAG_MANAGER_ID } from "../settings/clientSettings.js"
+
+export const GTMScriptTags = ({ gtmId }: { gtmId: string }) => {
+    if (!gtmId || /["']/.test(gtmId)) return null
+    return (
+        <>
+            <script
+                dangerouslySetInnerHTML={{
+                    __html: `/* Prepare Google Tag Manager */
+window.dataLayer = window.dataLayer || [];
+function gtag(){dataLayer.push(arguments);}
+gtag("consent","default",{"ad_storage":"denied","analytics_storage":"denied","wait_for_update":1000});
```

Comment:
The `wait_for_update: 1000` is an interesting dilemma here:
Ideally we would indicate to GTM at the very start already whether the user has consented or not; however we won't know this until our OWID scripts have loaded and fire the `consent, update` event.

This line tells GTM to withhold all events (incl. pageload events) for the first 1000ms after pageload, or until the `consent, update` event comes in (whichever happens first), so it can send the first few events with the tracking cookie information already if consent was granted.